pinot-segment-local/src/main/java/org/apache/pinot/segment/local/function/GroovyFunctionEvaluator.java

```diff
@@ -86,13 +86,13 @@ public List<String> getArguments() {
   public Object evaluate(GenericRow genericRow) {
     for (String argument : _arguments) {
       Object value = genericRow.getValue(argument);
-      if (value == null) {
-        // FIXME: if any param is null a) exit OR b) assume function handles it ?
-        return null;
-      }
       _binding.setVariable(argument, value);
     }
-    return _script.run();
+    try {
+      return _script.run();
+    } catch (Exception e) {
```

Comment:
wont errors we were previously catching and throwing during record transformation just get hidden now?